src/Okta.Sdk/Model/VerifyUserFactorResponse.cs

```diff
@@ -79,7 +79,7 @@ public bool ShouldSerializeEmbedded()
         /// Gets or Sets Links
         /// </summary>
         [DataMember(Name = "_links", EmitDefaultValue = true)]
-        public LinksSelf Links { get; set; }
+        public VerifyUserFactorResponseLinks Links { get; set; }
```

Comment:
Is this a breaking change?

Reply:
It is broken in current 7.0.0 branch, since the VerifyUserFactorResponse doesn't corresponds with the answer from the api.
So imho it's not a breaking change, but a bugfix.

Reply:
This is one of those gray scenarios where you are technically making interface changes, but the original behavior never worked. In this case, I'm inclined to categorize this as a bug fix because:

* `VerifyUserFactorResponseLinks` is a superset of `LinksSelf`. `LinkSelf` exposed a Self property only, while `VerifyUserFactorResponseLinks` exposes `Self`, `Cancel` and `Poll`. So, if you had the following code line in your program, it won't break after this change:
```
var link = response.Links.Self.Href;
```
* As @3sRykaert mentioned, the response.Links property was unreliable because it didn't fully correspond with the server's response. 

Thoughts @bryanapellanes-okta ?

Reply:
> This is one of those gray scenarios where you are technically making interface changes, but the original behavior never worked. In this case, I'm inclined to categorize this as a bug fix because:
> 
> * `VerifyUserFactorResponseLinks` is a superset of `LinksSelf`. `LinkSelf` exposed a Self property only, while `VerifyUserFactorResponseLinks` exposes `Self`, `Cancel` and `Poll`. So, if you had the following code line in your program, it won't break after this change:
> 
> ```
> var link = response.Links.Self.Href;
> ```
> 
> * As @3sRykaert mentioned, the response.Links property was unreliable because it didn't fully correspond with the server's response.
> 
> Thoughts @bryanapellanes-okta ?

I suspected it was a scenario as described, 👍